terraform/install.sh

```diff
@@ -157,6 +159,7 @@ createProject() {
       log "If you don't have access to this folder, please make sure to request at:"
       log "go/experimental-folder-access"
       log ""
+      python telemetry.py event=googler-project project=$project_id version=$VERSION
```

Comment:
Either the name is not descriptive or you will have a hard time knowing from the telemetry that this event instance is associated with creating a new project

Reply:
I was thinking of storing a separate file with constants, where I could store brief variable names, longer descriptions, and numbers corresponding to events. Do you think this would be a better way to do it, or make it unnecessarily complex?